src/core/server.cc

```diff
@@ -75,7 +74,7 @@ class ScopedAtomicIncrement {
 // InferenceServer
 //
 InferenceServer::InferenceServer()
-    : ready_state_(ServerReadyState::SERVER_INVALID), next_request_id_(1)
+    : ready_state_(ServerReadyState::SERVER_INVALID)
```

Comment:
Is there a reason you don't set this as the default assignment in the header file?

Reply:
Stylistically we don't typically use default assignment.